modules/core/src/main/scala/com.fortysevendeg.hood/json/JsonService.scala

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
Strange package `modules/core/src/main/scala/com.fortysevendeg.hood/json/JsonService.scala` 🤔

Reply:
Yeah, IntelliJ can be tricky about those. Fixed but also increased the number of files to review... sorry about it!